docs/how-to-guides/advanced-user-guides/enable-api-audit-log.md

```diff
@@ -51,6 +51,8 @@ Audit log policies allow end users to configure redactions using `AuditPolicy` c
 
 All configured audit log policies are additive.
 
+Redaction policies for headers use a regex engine to redact headers, while a JsonPath engine is used to redact request/response headers.
```

Comment:
we'll probably want to expand upon the JSONPath here. At the least we can provide a link to https://goessner.net/articles/JsonPath/ and mention that our engine does not support the script or filter expressions.